airgun/entities/host_new.py

```diff
@@ -480,6 +480,8 @@ def get_virtualization(self, entity_name):
         view = self.navigate_to(self, 'NewDetails', entity_name=entity_name)
         view.wait_displayed()
         self.browser.plugin.ensure_page_safe()
+        view.details.virtualization.read()
+        time.sleep(3)
```

Comment:
I think this is unnecessary, can you please explain why you need hard sleep?

Reply:
The trick is, the page only loads after calling view.details.virtualization.read(), and I need to wait at least 1 or 2 seconds for the card to load; it noticibly takes a few seconds before it pops up. So the workaround is to navigate there and try to read, wait, and then actually return a read

Reply:
I understand you need to add the sync login to the read() method.  Override the read to add that sync logic using wait for